<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBConnectDlg</class>
 <widget class="QDialog" name="DBConnectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConnectionConfig</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>401</width>
     <height>224</height>
    </rect>
   </property>
   <property name="title">
    <string>数据库连接设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="1">
     <widget class="QLineEdit" name="ledtUserName"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>驱动：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cbDriverList"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="ledtHostName"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>主机名：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>端口：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="ledtPassword"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>数据库名：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="ledtDatabaseName"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="ledtPort"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblConnectHint">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTestConnection">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAccept">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReject">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
